clients/src/main/java/org/apache/kafka/clients/producer/internals/TransactionManager.java

```diff
@@ -348,7 +348,10 @@ private TransactionalRequestResult beginCompletingTransaction(TransactionResult
                             .setTransactionalId(transactionalId)
                             .setProducerId(producerIdAndEpoch.producerId)
                             .setProducerEpoch(producerIdAndEpoch.epoch)
-                            .setCommitted(transactionResult.id));
+                            .setCommitted(transactionResult.id),
+                    true,
```

Comment:
I think in this PR we can mark EndTxn stable so this value doesn't need to be true.